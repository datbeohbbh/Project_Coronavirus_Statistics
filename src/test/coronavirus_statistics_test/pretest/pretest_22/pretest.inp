31 30
q 1 26 197
q 3 26 196
q 3 26 1
q 1 26 380
q 1 3 36
q 1 26 293
q 2 26 296
q 1 6 218
q 1 23 218
q 1 8 275
q 3 3 4
q 1 25 168
q 1 13 459
? 6
? 4 2
q 1 20 454
q 2 6 200
q 1 10 395
? 7
q 2 25 53
q 1 24 22
q 1 27 11
q 1 12 120
? 4 6
? 4 4
? 6
? 7
? 4 3
? 6
? 6