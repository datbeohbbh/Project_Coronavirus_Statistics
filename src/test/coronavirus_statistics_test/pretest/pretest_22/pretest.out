Query ? 6 :
The city with the highest recovery is city with the index 26
Quantity: 296 people

Query ? 4 2 :
The answer is the city with the index 25 
- Information of the city 25 -
-> Number of current patients : 168
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 26
Quantity: 197 people

Query ? 4 6 :
The answer is the city with the index 12 
- Information of the city 12 -
-> Number of current patients : 120
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 32
-> Has recovered : 0
-> Was dead : 4

Query ? 6 :
The city with the highest recovery is city with the index 26
Quantity: 296 people

Query ? 7 :
The city with the highest death is city with the index 26
Quantity: 197 people

Query ? 4 3 :
The answer is the city with the index 24 
- Information of the city 24 -
-> Number of current patients : 22
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 26
Quantity: 296 people

Query ? 6 :
The city with the highest recovery is city with the index 26
Quantity: 296 people

